Gemfile.lock

```diff
@@ -347,10 +359,10 @@ GEM
     json-schema-rspec (0.0.4)
       json-schema (~> 2.5)
       rspec
-    jsonpath (0.8.11)
+    jsonpath (0.9.5)
       multi_json
       to_regexp (~> 0.2.1)
-    jwt (1.5.6)
+    jwt (2.1.0)
```

Comment:
Looks like this update breaks our OpenID Connect implementation.

Reply:
Do you think that's the cause of the test failure and not what I was writing about?

Reply:
I only looked at the rspec failures, your explanation for the cucumber one sounds plausible :)

Reply:
I'm not sure if downgrading is an option. Maybe opening a separate PR to upgrade?

Reply:
Depends on how big the fix is, if it's a one or two liner do as you prefer, I'd be fine to include it here too.